tests/test_extensions.py

```diff
@@ -0,0 +1,132 @@
+"""
+Unit tests for Extension Manager
+"""
+
+import sys
+import os
+import unittest
+import mock
+import shutil
+
+dir_of_executable = os.path.dirname(__file__)
```

Comment:
Even if you didn't like `tox` you would run the tests with `nosetest` or `pytest` because you don't want to run a single test file. You would want to test all the test files to make sure nothing is broken. 

>It's only a line. :)

Technically 4 lines 😉 . But it's about consistency through out the project.